.travis.yml

```diff
@@ -15,12 +13,15 @@ matrix:
         # Lint the first commit in the PR.
         - \[ -z "$TRAVIS_COMMIT_RANGE" \] || (echo -e '\nLinting the commit message according to the guidelines at https://goo.gl/p2fr5Q\n' && git log $TRAVIS_COMMIT_RANGE --pretty=format:'%h' --no-merges | tail -1 | xargs npx -q core-validate-commit --no-validate-metadata)
     - name: "Test Suite"
+      addons:
+        apt:
+          sources:
+            - ubuntu-toolchain-r-test
+          packages:
+            - g++-4.9
       install:
+        - export CC='ccache gcc-4.9' CXX='ccache g++-4.9' JOBS=2
         - ./configure
         - make -j2 V=
       script:
-        - make -j2 test
-      before_install:
-        - export CXX="ccache clang++ -Qunused-arguments"
-        - export CC="ccache clang -Qunused-arguments -Wno-unknown-warning-option"
-        - export JOBS=2
+        - PARALLEL_ARGS='--flaky-tests=skip' make -j1 test
```

Comment:
Please keep using `make -j2` unless we know that not doing so is actually causing us trouble

Reply:
This has been causing us trouble. And as of b3ae9150de99a9b7b92ceddb11cd9e43ea92c7cb is on par with our CI settings.

Reply:
@addaleax 

> My impression would be that this means that `make -jN` is potentially broken everywhere? 

It is for me. The breakage is even easier to reproduce with V8 7.1 (because more actions are executed with GYP).
Relanding https://github.com/nodejs/node/pull/23257 would fix it, but breaks rebuilds on macOS (see https://github.com/nodejs/node/pull/23423#issuecomment-432241209)

Reply:
@refack If this is accurate (i.e. this is only a temporary workaround), feel free to add me as a reviewer. I’d appreciate having some mention of why we do this, both the compiler change and the `-j1` parts, in the commit message, if that’s okay with you.

Reply:
We should reconsider after we solve the make cyclic dependency.
(Again IMHO `make test -j2` doesn't give use much benefit in terms on time, maybe otherwise.)